IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1044,133 +1027,40 @@ sub _create_parchment_page {
-</body>
-</html>
-
-EOF
-        ;
-
-    my $html_file = $self->content_directory->file('index.html');
-    my $html_fh   = $html_file->openw;
-    $html_fh->binmode(':utf8');
-
-    print $html_fh $html;
-
-}
-
-sub _create_quixe_page {
-    my $self = shift;
-
-    my $title = $self->title;
-
-    # Search the content directory for the I7 file to link to.
-    my $i7_file = $self->inform_game_file;
-
-    unless ($i7_file) {
-        die "Could not find an I7 file in this entry's content directory.";
-    }
-
-    # Create a JS file for this game.
-    my $js_file = Path::Class::File->new( $i7_file . '.js' );
-    $self->_make_js_file( $i7_file, $js_file );
-
-    my $js_filename = $js_file->relative( $self->content_directory );
-
-    my $entry_id = $self->id;
-
-    my $tag_text = $self->interpreter_tag_text;
-    my $html     = <<EOF
-<!DOCTYPE html>
-<html>
-<head>
-<title>IFComp &ndash; $title &ndash; Play</title>
-
-<meta name="viewport" content="width=device-width, user-scalable=no">
-
-$tag_text
-
-<style type="text/css">
-
-body {
-  margin: 0px;
-  height: 100%;
-}
-
-#gameport {
-  position: absolute;
-  overflow: hidden;
-  width: 100%;
-  height: 100%;
-  background: #CCAA88;
-  margin: 0px;
-}
-
-</style>
-
-<script type="text/javascript">
-game_options = {
-  use_query_story: false,
-  set_page_title: true,
-  recording_url: '/play/$entry_id/transcribe',
-  recording_label: '$title',
-  recording_format: 'simple'
-};
-</script>
-
-<script src="$js_filename" type="text/javascript"></script>
-
+    <meta charset="utf-8">
+    <title>IFComp &ndash; $title &ndash; Play</title>
+    <meta name="viewport" content="width=device-width,user-scalable=no">
+    <script src="/static/interpreter/jquery.min.js"></script>
+    <script src="/static/interpreter/ie.js" nomodule></script>
+    <script src="/static/interpreter/main.js" type="module"></script>
+    <link rel="stylesheet" href="/static/interpreter/web.css">
+    <script>
+        parchment_options = {
+            default_story: [ "$i7_file" ],
+            recording_url: '/play/$entry_id/transcribe',
+            recording_label: '$title',
+            recording_format: 'simple'
+        }
+    </script>
 </head>
 <body>
-
-<div id="gameport">
-<div id="windowport">
-<noscript><hr>
-<p>You'll need to turn on Javascript in your web browser to play this game.</p>
-<hr></noscript>
-</div>
-<div id="loadingpane">
-<img src="/static/interpreter/glkote/waiting.gif" alt="LOADING"><br>
-<em>&nbsp;&nbsp;&nbsp;Loading...</em>
-</div>
-<div id="errorpane" style="display:none;"><div id="errorcontent">...</div></div>
-</div>
-
+    <div id="gameport">
+        <div id="windowport">
+            <noscript>
+                <p>You'll need to turn on Javascript in your web browser to play this game.</p>
+            </noscript>
+        </div>
+        <div id="loadingpane">
+            <img src="/static/interpreter/waiting.gif" alt="LOADING"><br>
```

Comment:
Just had an idea: it shouldn't be hard to make it show the entry's cover image here, but I don't know what the substitution would be. Should I make a new issue?